ftw/linkchecker/linkchecker.py

```diff
@@ -0,0 +1,100 @@
+from multiprocessing import Pool
+import time
+import requests
+import urls_testing
+import os
+import sys
+
+urls = urls_testing.urls
+number_of_processes = len(urls)
+timeout = 1
```

Comment:
Either use a constant by using capitals:
```python
TIMEOUT = 1
```
or remove it here ind use a static `1` in the call.

Reply:
I removed the variable and directly defined the timeout in context.

Reply:
I would put it in a configuration file at some point

Reply:
Ok, I put it in a constant. That enables us to later load it from any config file easily.